apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -240,10 +240,10 @@ class LivechatClass {
 		session: ClientSession,
 	): Promise<{ room: IOmnichannelRoom; closedBy: ChatCloser; removedInquiry: ILivechatInquiryRecord | null }> {
 		const { comment } = params;
-		const { room } = params;
+		const { room, forceClose } = params;
```

Comment:
I wonder if we should re-check the setting here :thinking: 
I know you're already checking in the endpoint level, but maybe we should have something at a lower level to prevent force closing if the setting is disabled. This is more of an opinion, so I'd like to hear what you think or if you thought about this already

Reply:
Hmmmm, not sure.

I mean, there's no other place that we expose that can use this param right now so thought checking at the caller makes sense.

Wdyt?

Reply:
hmmm got your point. but can we have a third opinion? I'd like to hear @MarcosSpessatto 
I thought that having the check at this level too could help us not be caught off guard in case this new prop is used without the setting check in our codebase later.

Reply:
Yeah, i got your point. But keep in mind we don't expose the closing mechanism to any place that a user can interact with. Apps are through a bridge we control, cronjobs are code we control. The only place that allows for user intervention are the 2 room closing endpoints, and one is for visitors and only allows to receive a roomId.

So in any possible scenario, we should be safe by letting the endpoint do what it wants to do.

Reply:
I understand @matheusbsilva137 point, and it indeed makes sense. But I think we can go with Kevin's approach, in the future, if we need to use that somewhere else, then we could refactor, moving the validation to the lowest level.